Jenkinsfile.yml

```diff
@@ -4,6 +4,9 @@ node('ubuntu18.04-OnDemand'){
         'REPO_PRI=https://github.com/secure-device-onboard/pri',
         'REPO_SCT=https://github.com/secure-device-onboard/supply-chain-tools',
         'REPO_IOT=https://github.com/secure-device-onboard/iot-platform-sdk',
+        "TEST_DIR=${WORKSPACE}/sdo-test",
+        "MANUFACTURER_DB_CONNECT_STRING=jdbc:mariadb://127.0.0.1:3306/sdo",
```

Comment:
AIO smoketest doesn't use MANUFACTURER_DB_CONNECT_STRING and RESELLER_DB_CONNECT_STRING variables. They can be removed from here.

Reply:
Removed the variables from the jenkinsfile